kuma/api/v1/views.py

```diff
@@ -245,6 +245,12 @@ def search(request, locale):
                           Q('term', locale={'value': locale, 'boost': 8}),
                       ])))
 
+    # Add excerpts with search results highlighted
+    search = search.highlight(*WikiDocumentType.excerpt_fields)
+    search = search.highlight_options(order='score',
+                                      pre_tags=['<mark>'],
+                                      post_tags=['</mark>'])
```

Comment:
As mentioned in the general comment, I think this should be `<b>` and `</b>`. But feel free to ignore if you disagree.